### PR TITLE
Restore sysbench documents

### LINKED ISSUE #13825: Restore sysbench documents
## Documents

The `sysbench` testing references were deleted in https://github.com/apache/shardingsphere/pull/13005. 
And the `performance-test` was accidentally renamed to `sysbench-test` in https://github.com/apache/shardingsphere/pull/13182. 

Consider restoring the `sysbench` documents and update the configurations to the latest.

### PR DESCRIPTION
**Pull request by Swastyy**
Fixes #13825 

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

File: docs/document/content/reference/test/performance-test/sysbench-test.en.md [hunk 1]
```diff
@@ -1,435 +1,1425 @@
...
+ShardingSphere-Proxy * 1: ${host-proxy}
+MySQL Server * 2: ${host-mysql-1}, ${host-mysql-2}
+```
+
+The hardware standards of Jenkins and Sysbench machines can appropriately lower.
+
+#### Software Environment
+
+```
+Jenins: The latest version
```
Comment (TeslaCN) [inline docs/document/content/reference/test/performance-test/sysbench-test.en.md:20]:
```suggestion
Jenkins: The latest version
```

File: docs/document/content/reference/test/performance-test/sysbench-test.en.md [hunk 2]
```diff
@@ -1,435 +1,1425 @@
...
+  - sharding@:sharding
+props:
+  max-connections-size-per-query: 10
+  kernel-executor-size: 128  # Infinite by default.
+  proxy-frontend-flush-threshold: 128  # The default value is 128.
+  proxy-opentracing-enabled: false
+  proxy-hint-enabled: false
+  sql-show: false
+  check-table-metadata-enabled: false
+  lock-wait-timeout-milliseconds: 50000 # The maximum time to wait for a lock
```
Comment (TeslaCN) [inline docs/document/content/reference/test/performance-test/sysbench-test.en.md:289]: The property `lock-wait-timeout-milliseconds` has been removed.

File: docs/document/content/reference/test/performance-test/sysbench-test.en.md [hunk 3]
```diff
@@ -1,435 +1,1425 @@
...
+
+#### Master branch version
+
+server.yaml
 
 ```yaml
-schemaName: sharding_db
+users:
+  - root@%:root
+  - sharding@:sharding
```
Comment (TeslaCN) [inline docs/document/content/reference/test/performance-test/sysbench-test.en.md:280]: Could you please check this configuration again?

**Review by TeslaCN — CHANGES_REQUESTED** [review]

File: docs/document/content/reference/test/performance-test/sysbench-test.en.md (hunk 3 above)
Comment (Swastyy) [reply docs/document/content/reference/test/performance-test/sysbench-test.en.md:280]: Okay, I am doing.

Comment (Swastyy) [reply docs/document/content/reference/test/performance-test/sysbench-test.en.md:280]: Hi @TeslaCN , sorry for late reply. Please tell me from where can I check the correct configurations.

Comment (TeslaCN) [reply docs/document/content/reference/test/performance-test/sysbench-test.en.md:280]: @Swastyy The `server.yaml` can be found here
https://github.com/apache/shardingsphere/blob/master/shardingsphere-proxy/shardingsphere-proxy-bootstrap/src/main/resources/conf/server.yaml

Comment (Swastyy) [reply docs/document/content/reference/test/performance-test/sysbench-test.en.md:280]: Done.

Comment (TeslaCN) [reply docs/document/content/reference/test/performance-test/sysbench-test.en.md:280]: The configuration of users has been to the following format:
```
rules:
  - !AUTHORITY
    users:
      - root@%:root
      - sharding@:sharding
    provider:
      type: ALL_PRIVILEGES_PERMITTED
```

Comment (TeslaCN) [reply docs/document/content/reference/test/performance-test/sysbench-test.en.md:280]: You may need to make sure the configuration in this documents can be used to start up a latest ShardingSphere-Proxy.

**Review by TeslaCN — APPROVED** [review]
@Swastyy Thank you.

**Pull request merged**